src/Console/Command/DebugWeavingCommand.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/*
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2015, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\Console\Command;
+
+use Go\Instrument\ClassLoading\CachePathManager;
+use Go\Instrument\ClassLoading\CacheWarmer;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Console command for querying an information about aspects
+ *
+ * @codeCoverageIgnore
+ */
+class DebugWeavingCommand extends BaseAspectCommand
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected function configure()
+    {
+        parent::configure();
+        $this
+            ->setName('debug:weaving')
+            ->setDescription('Checks consistency in weaving process')
+            ->setHelp(<<<EOT
+Allows to check consistency of weaving process, detects circular references and mutual dependencies between
+subjects of weaving and aspects.
+EOT
+            );
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->loadAspectKernel($input, $output);
+
+        $io = new SymfonyStyle($input, $output);
+
+        $io->title('Weaving debug information');
+
+        $cachePathManager = $this->aspectKernel->getContainer()->get('aspect.cache.path.manager');
+        $warmer = new CacheWarmer($this->aspectKernel, new NullOutput());
+        $warmer->warmUp();
+
+        $proxies = $this->getProxies($cachePathManager);
+
+        $cachePathManager->clearCacheState();
+        $warmer->warmUp();
+
+        $errors = 0;
+
+        foreach ($this->getProxies($cachePathManager) as $path => $content) {
+            if (!isset($proxies[$path])) {
+                $io->writeln(sprintf('<fg=white;bg=red;options=bold>[ERR]</>: Proxy on path "%s" is generated on second "warmup" pass.', $path));
```

Comment:
Maybe just switch to `$io->error()` or `$io->caution()` here? Instead of manual formatting?